dream_layer_backend/comfy_nodes/api_nodes/runway_text2img.py

```
@@ -0,0 +1,326 @@
+                    
+                    # Gradually increase poll interval to reduce API calls
+                    poll_interval = min(poll_interval * 1.2, max_poll_interval)
+                    
+                else:
+                    # Unknown status
+                    print(f"[RunwayText2Img] Unknown status: {status}, continuing to poll...")
+                    time.sleep(poll_interval)
+                    
+            except httpx.HTTPStatusError as e:
+                if e.response.status_code == 404:
+                    raise RuntimeError(f"Task {task_id} not found")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)